datastore/elastic_search/query.py

```diff
@@ -235,7 +287,72 @@ def _generate_es_ngram_search_dictionary(entity_name, ngrams_list, fuzziness_thr
         },
         'number_of_fragments': 20
     }
+    return data
```

Comment:
if '_generate_es_ngram_search_dictionary' is no longer needed then please remove it.